Assets/Scripts/Internal/DaggerfallBillboard.cs

```diff
@@ -439,6 +439,27 @@ public void OnDrawGizmos()
                 }
             }
         }
+        
+        /// <summary>
+        /// Draws a circle at the bottom of the bilboard to make it easier to judge the size regardless of rotation.
+        /// </summary>
+        public void OnDrawGizmosSelected()
```

Comment:
`OnDrawGizmosSelected` is meant to be private (there are exceptions ofc, but justified)